### ISSUE TITLE
Need to implement new way of getting price guides and product list, the current API calls will be removed by December 2024

### ISSUE THREAD

**Issue by tomasjanak** [alexander-pick/MKMTool#68]
Cardmarket has change the way to get price guides and more importantly the product list  https://news.cardmarket.com/en/Magic/were-making-the-price-guide-and-product-catalogue-available-for-download. Without updated product list, most features of MKMTool will stop working for new cards.
Unfortunately, not only the has the way of getting the files changed, their format has changed as well. Lot of MKMTool code would have to be rewritten.

**Comment by MeggaZone** [comment]
hello man, i love your project but i'm still not able to use it, i can't find my api token on my profile... it says i'm not able to get it, do you have any suggestion?

**Comment by tomasjanak** [comment]
API access has been restricted to professional sellers for the past 3 years (see https://github.com/alexander-pick/MKMTool/issues/63) and not even to all of them, they gave it only on a case by case basis. They also claim that they are no longer giving it to anyone (from https://help.cardmarket.com/en/cardmarket-api: "Currently, we are not accepting applications for access to the Cardmarket API.")
So basically if you did not use the tool until now, you can't use it anymore until cardmarket changes their policy.
